EpikCord/client/commands.py

```diff
@@ -49,17 +52,455 @@ def to_dict(self):
+            name,
+            description,
+            name_localizations=name_localizations,
+            description_localizations=description_localizations,
+            required=required,
+            type=ApplicationCommandOptionType.BOOLEAN,
+        )
+
+class UserOption(BaseApplicationCommandOption):
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        *,
+        name_localizations: List[Localization] = [],
+        description_localizations: List[Localization] = [],
+        required: bool = False,
+    ):
+        super().__init__(
+            name,
+            description,
+            name_localizations=name_localizations,
+            description_localizations=description_localizations,
+            required=required,
+            type=ApplicationCommandOptionType.USER,
+        )
+
+class RoleOption(BaseApplicationCommandOption):
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        *,
+        name_localizations: List[Localization] = [],
+        description_localizations: List[Localization] = [],
+        required: bool = False,
+    ):
+        super().__init__(
+            name,
+            description,
+            name_localizations=name_localizations,
+            description_localizations=description_localizations,
+            required=required,
+            type=ApplicationCommandOptionType.ROLE,
+        )
+
+class AttachmentOption(BaseApplicationCommandOption):
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        *,
+        name_localizations: List[Localization] = [],
+        description_localizations: List[Localization] = [],
+        required: bool = False,
+    ):
+        super().__init__(
+            name,
+            description,
+            name_localizations=name_localizations,
+            description_localizations=description_localizations,
+            required=required,
+            type=ApplicationCommandOptionType.ATTACHMENT,
+        )
+
+class MentionableOption(BaseApplicationCommandOption):
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        *,
+        name_localizations: List[Localization] = [],
+        description_localizations: List[Localization] = [],
+        required: bool = False,
+    ):
+        super().__init__(
+            name,
+            description,
+            name_localizations=name_localizations,
+            description_localizations=description_localizations,
+            required=required,
+            type=ApplicationCommandOptionType.MENTIONABLE,
+        )
+
+class ChannelOption(BaseApplicationCommandOption):
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        *,
+        name_localizations: List[Localization] = [],
+        description_localizations: List[Localization] = [],
+        required: bool = False,
+        channel_types: Optional[List[ChannelType]]
```

Comment:
Optional not initialized to None?

```suggestion
        channel_types: Optional[List[ChannelType]] = None
```